flatpak/org.jabref.jabref.json

```
@@ -0,0 +1,33 @@
+{
+    "app-id" : "org.jabref.jabref",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "JabRef",
+    "modules" : [
+        {
+            "name" : "JabRef",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "tar -xvzf flatpak/JabRef-portable_linux.tar.gz -C /app/"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "path": "JabRef-portable_linux.tar.gz"
+                }
```

```suggestion
                },
                {
                    "type": "file",
                    "path": "org.jabref.jabref.desktop"
                }
```

Remember to rename jabref.desktop to org.jabref.jabref.desktop as per flatpak rules